mk/dependabot.mk

```diff
@@ -0,0 +1,27 @@
+SHELL:=/bin/bash
```

Comment:
Could you also document how to run these commands?

Reply:
This is the way the SHELL can be defined, the $${dir:1} requires to be executed with /bin/bash that's why it's defined here. See [this link](https://www.gnu.org/software/make/manual/html_node/Choosing-the-Shell.html) for more details

Reply:
Sorry, I meant in the PR description, how to execute the commands, for further reference